ui/src/hosts/containers/HostPage.js

```diff
@@ -205,6 +202,32 @@ class HostPage extends Component {
       </div>
     )
   }
+
+  get dashboardLinks() {
+    const {
+      params: {sourceID},
+    } = this.props
+    const {hosts} = this.state
+
+    if (!sourceID || !hosts) {
+      return []
+    }
+
+    return Object.values(hosts).map(({name}) => ({
+      id: name,
+      text: name,
```

Comment:
does this not break if two hosts have the same name, or is that protected against that?

Reply:
It will, but my thinking was that the `name` _is_ the unique identifier for a host (both conceptually and in terms of the actual data we have available to us). So they are treated a little bit differently than dashboards (which cannot be identified by name).